src/graphql/data/SearchTokens.ts

```diff
@@ -87,7 +99,7 @@ export function useSearchTokens(searchQuery: string, chainId: number) {
     data?.searchTokens?.forEach((token) => {
       if (token.project?.id) {
         const existing = selectionMap[token.project.id]
-        if (isMoreRevelantToken(token, existing, searchChain)) selectionMap[token.project.id] = token
```

Comment:
it looks like there's a behavior change here that might not be intentional--in the old one there were cases where `selectionMap[token.project.id]` wouldn't be assigned a value here, but now every iteration assigns a value. Is that intended?

Reply:
In cases where `isMoreRevelantToken` would've returned false, `dedupeCrosschainTokens` returns `existing`, so this is just a refactor for readability